apps/demos/Demos/DropDownBox/MultipleSelection/description.md

```diff
@@ -0,0 +1,8 @@
+DropDownBox is an advanced drop-down editor that allows you to embed any content, including other components.
+
+DevExtreme offers multiple drop-down editor options. To find out which editor you should use, review the following article: [How to Choose a Drop-Down Editor](/Documentation/Guide/UI_Components/Lookup/Choose_a_Drop-Down_Editor/).
+
+If the UI component embedded into the DropDownBox allows multiple selection, synchronize the DropDownBox value with the selected items. [Synchronization instructions](/Demos/WidgetsGallery/Demo/DropDownBox/SingleSelection/) are the same for every selection mode.
```

Comment:
First question I'm having at this point -- can I embed only one UI component into the `DropDownBox` component's drop-down window?

> [Synchronization instructions](/Demos/WidgetsGallery/Demo/DropDownBox/SingleSelection/) are the same for every selection mode.

Of what selection modes do we speak? It's not obvious from my standpoint.

This paragraph could have looked as follows:

```suggestion
If the UI component embedded into the DropDownBox component's drop-down window allows multiple selection, synchronize the component's value with the state of selected items as demonstrated in the following topic: [Synchronization instructions](/Demos/WidgetsGallery/Demo/DropDownBox/SingleSelection/).
```

However, you can probably come up with something better than that.